### DISCUSSION TITLE
Install libraries python

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#834] [Q&A] [answered]
Dear Algoritma Team,

Saya punya permasalahan pada saat install library stldecompose. Awalnya saya lakukan install dengan cara !pip install stldecompose hasilnya sukses. 

<img width="839" alt="image" src="https://github.com/teamalgoritma/community/assets/103427989/c83bd25a-3771-4d4a-bf1b-b072024d2e9b">

Namun saat saya menjalankan syntax from stldecompose import decompose, forecast hasilnya error. Kemudian saya coba menginstal library melalui env, hasilnya sukses.
capture screen
<img width="676" alt="image" src="https://github.com/teamalgoritma/community/assets/103427989/80d5fe39-4fc4-4448-a34a-7e893ebe085f">

Namun saat saya menjalankan syntax from stldecompose import decompose, forecast
hasilnya masih error.
<img width="875" alt="image" src="https://github.com/teamalgoritma/community/assets/103427989/4f7c0a8a-0490-4bb4-a40c-0cfdfefb6eaf">

Kondisi ini juga terjadi ketika saya melakukan install library statsmodels

Mohon sarannya,
Best regards
JT

**Comment by jafarhf1** [comment, marked as answer]
Dear @joejnt ,

Mengenai pertanyaanya bisa dicoba menggunakan solusi dari tautan berikut: https://stackoverflow.com/questions/62013312/importing-stldecompose-result-in-error-cannot-import-name-maybe-get-pandas-w

Semoga membantu

Best regards,
Team Algoritma

**Reply by joejnt** [reply]
Dear Team Algoritma

Saya sudah mencoba dengan melakukan pip install statsmodels==0.10.2, namun masih terjadi error. Saat ini saya sedang mencoba untuk melakukan conda install -c conda-forge statsmodels (https://www.statsmodels.org/stable/install.html). Mohon sarannya, untuk proses install tersebut sebaiknya melalui anaconda prompt dengan mengaktifkan env yang saya buat atau dapat dilakukan di jupyter notebook nya ?

Best regards,
JT
